Modules/posixmodule.c

```diff
@@ -5278,6 +5278,16 @@ parse_posix_spawn_flags(PyObject *setpgroup, int resetids, PyObject *setsigmask,
 #endif
     }
 
+    if (use_vfork) {
+#ifdef POSIX_SPAWN_USEVFORK
+        all_flags |= POSIX_SPAWN_SETSCHEDULER;
+#else
+        PyErr_SetString(PyExc_NotImplementedError,
```

Comment:
You can use argument_unavailable_error() here.

Reply:
I discussed with Pablo and using "posix_spawn()" function name is less useless than mentioning the "POSIX_SPAWN_USEVFORK flag".